target/scripts/startup/setup.d/postfix.sh

```diff
@@ -109,29 +109,31 @@ function _setup_postfix_late() {
 function __postfix__setup_override_configuration() {
   __postfix__log 'debug' 'Overriding / adjusting configuration with user-supplied values'
 
-  if [[ -f /tmp/docker-mailserver/postfix-main.cf ]]; then
-    cat /tmp/docker-mailserver/postfix-main.cf >>/etc/postfix/main.cf
+  local OVERRIDE_CONFIG_POSTFIX_MAIN='/tmp/docker-mailserver/postfix-main.cf'
+  if [[ -f ${OVERRIDE_CONFIG_POSTFIX_MAIN} ]]; then
+    cat "${OVERRIDE_CONFIG_POSTFIX_MAIN}" >>/etc/postfix/main.cf
```

Comment:
The name suggests an override, but we append - intended?

Reply:
Oh.. `OVERRIDE_` as a prefix for the var introduced here is just due to the intent, like mentioned with docs. We refer to these as Postfix config overrides, so it shares that same prefix with the master config too.